### PR TITLE
Feat 725 user logo and image

### PR DESCRIPTION
**Pull request by danielsh28**

### CODE REVIEW

File: src/components/molecules/Header.tsx [hunk 1]
```diff
@@ -1,8 +1,7 @@
 import React, { FC, useEffect } from 'react';
 import { observer } from 'mobx-react-lite';
 import { AppBar, Logo } from '../atoms';
-import AnywayImage from '../../assets/anyway.png';
-import { SignInIcon } from '../atoms/SignInIcon';
+
```
Comment (yuvalbl) [inline src/components/molecules/Header.tsx:4]: why the empty line?

File: src/components/molecules/UserProfileHeader.tsx [hunk 2]
```diff
@@ -19,15 +21,20 @@ const useStyles = makeStyles((theme) => ({
   welcomeMsg: {
     padding: theme.spacing(1),
   },
+  avatar: {
+    width: theme.spacing(5),
```
Comment (yuvalbl) [inline src/components/molecules/UserProfileHeader.tsx:25]: do not use `theme.spacing` for height / width
See `fontSize`
https://material-ui.com/api/icon/

File: src/components/molecules/UserProfileHeader.tsx [hunk 3]
```diff
@@ -19,15 +21,20 @@ const useStyles = makeStyles((theme) => ({
...
+    height: theme.spacing(5),
+  }
 }));
 
 interface IUserProfileHeader {
-  userDetails: IFormInput;
+  userDetails: ActualiUserInfo;
```
Comment (yuvalbl) [inline src/components/molecules/UserProfileHeader.tsx:31]: Why adding `Actual` to interface name?

File: src/utils/LogoMap.ts [hunk 4]
```diff
@@ -0,0 +1,14 @@
+import AnywayImage from '../assets/anyway.png';
+import OrYarokLogo from '../assets/Or_Yarok.png';
+
+
+const orYarok = 'אור ירוק'
```
Comment (yuvalbl) [inline src/utils/LogoMap.ts:5]: use i18n

File: src/utils/LogoMap.ts [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+const none = null;
+export const anywayLogo = AnywayImage;
```
Comment (yuvalbl) [inline src/utils/LogoMap.ts:7]: why not
```
import AnywayLogo from '../assets/anyway.png';
export AnywayLogo

```

File: src/utils/LogoMap.ts [hunk 6]
```diff
@@ -0,0 +1,14 @@
+import AnywayImage from '../assets/anyway.png';
+import OrYarokLogo from '../assets/Or_Yarok.png';
+
+
+const orYarok = 'אור ירוק'
+const none = null;
```
Comment (yuvalbl) [inline src/utils/LogoMap.ts:6]: why setting a const to null? simply use null (which in this case, also not required)

File: src/utils/LogoMap.ts [hunk 7]
```diff
@@ -0,0 +1,14 @@
...
+export const anywayLogo = AnywayImage;
+
+const logoMap  : Map<string | null,string>  = new Map();
+logoMap.set(orYarok,OrYarokLogo);
+logoMap.set(none,AnywayImage);
```
Comment (yuvalbl) [inline src/utils/LogoMap.ts:11]: This is not the way we treat images. also it is not considered "utils"
see `MapIcon.tsx` as example

File: src/utils/utils.ts [hunk 8]
```diff
@@ -46,3 +46,4 @@ export const DEMO_ID = 999;
 export const showOnlyOperCards = process.env.REACT_APP_SHOW_ONLY_OPER_CARDS === 'true';
 
 export const INITIAL_CENTER = { lat: 32.0853, lng: 34.7818 };
+
```
Comment (yuvalbl) [inline src/utils/utils.ts:49]: changes in this file are not required

File: src/components/molecules/UserProfileHeader.tsx (hunk 2 above)
Comment (danielsh28) [reply src/components/molecules/UserProfileHeader.tsx:25]: I'm using  `Avatar` component,  see https://material-ui.com/components/avatars/#sizes

File: src/components/molecules/UserProfileHeader.tsx (hunk 3 above)
Comment (danielsh28) [reply src/components/molecules/UserProfileHeader.tsx:31]: that's an old mistake will be fixed

File: src/utils/LogoMap.ts (hunk 4 above)
Comment (danielsh28) [reply src/utils/LogoMap.ts:5]: actually, I  think that we need a drop-down for a fixed set of organizations.  your thoughts on this @yuvalbl ?

File: src/components/molecules/UserProfileHeader.tsx (hunk 2 above)
Comment (yuvalbl) [reply src/components/molecules/UserProfileHeader.tsx:25]: still,  using spacing for size seems like a bad practice (although appears in the docs)

File: src/utils/LogoMap.ts (hunk 4 above)
Comment (yuvalbl) [reply src/utils/LogoMap.ts:5]: This will be done on a different task.
Maybe it would be better to remove this until then

**Pull request merged**